apps/web/app/utils/authGG.ts

```diff
@@ -0,0 +1,39 @@
+import { signOut } from 'firebase/auth';
+import firebase from 'firebase/compat/app';
+
+import auth from './config';
+import { userProfileState } from './state';
+
+export const handleSignInWithGG = async () => {
+	const provider = new firebase.auth.GoogleAuthProvider();
+	try {
+		const result = await auth.signInWithPopup(provider);
+		const user = result.user;
+		if (user) {
+			const firebaseUserIdToken = await user.getIdToken();
+			console.log(firebaseUserIdToken);
+			const response = await fetch('http://localhost:3005/api/signin', {
```

Comment:
should load `http://localhost:3005/api/signin` from `.env` file, and make an `utils/config.ts` that have `environments` variable reflect whatever we needed from `.env`.